phoenix-core/src/main/java/org/apache/phoenix/iterate/ExplainTable.java

```diff
@@ -120,7 +120,7 @@ protected void explain(String prefix, List<String> planSteps) {
         } else {
             explainSkipScan(buf);
         }
-        buf.append("OVER ").append(tableRef.getTable().getPhysicalName().getString());
+        buf.append("OVER ").append(tableRef.getTable().getPhoenixPhysicalName().getString());
```

Comment:
Now here getPhysicalName -> getPhoenixPhysicalName. Why are we introducing a new name method?